SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.h

```diff
@@ -11,12 +11,13 @@ class FCodeWriter;
 struct FComponentIdGenerator;
 
 extern TArray<UClass*> SchemaGeneratedClasses;
-extern TMap<FString, FSchemaData> ClassPathToSchemaData;
+extern TMap<FString, FActorSchemaData> ActorClassPathToSchema;
```

Comment:
If touching global variables can they be removed?

Reply:
Or at least put in a namespace.

Reply:
That's outside of this PR unfortunately. I agree this file should be refactored.